src/node/mempool_persist.cpp

```diff
@@ -179,13 +182,16 @@ bool DumpMempool(const CTxMemPool& pool, const fs::path& dump_path, FopenFn mock
         file << version;
 
         if (!pool.m_opts.persist_v1_dat) {
-            std::vector<std::byte> xor_key(8);
-            FastRandomContext{}.fillrand(xor_key);
-            file << xor_key;
-            file.SetXor(xor_key);
+            std::vector<std::byte> xor_key_vector(8);
+            FastRandomContext{}.fillrand(xor_key_vector);
+            file << xor_key_vector;
+
+            uint64_t m_xor;
```

Comment:
Should not use `m_`-prefix for non-member variables.

Reply:
You're right, I've used `xor_key` in the same case before